test/decorateRequest.js

```diff
@@ -65,7 +65,7 @@ describe('decorateRequest', function() {
         it('returns err to host application for processing', function(done) {
           var app = express();
 
-          app.use(proxy('/reject-promise', {
+          app.use(proxy('httpbin.org', {
```

Comment:
Unrelated issue.  This test would always fail since host __must__ start with `http(s)`.

Reply:
I think this comment is incorrect.    Test pass today without all starting with http.     Current code assumes http if http/https is not provided.

Reply:
👍